src/App.js

```diff
@@ -37,51 +38,53 @@ function App() {
         setIsLoading(false);
       });
     }
-
     return unsubscribe;
   }, [token]);
 
   return (
     <div className="App">
       <header className="App-header">
-        <div style={{ background: '#fff', padding: '40px', borderRadius: 5 }}>
-          <Switch>
-            <Route
-              exact
-              path="/"
-              render={() =>
-                token ? (
-                  <Redirect to="/list" />
-                ) : (
-                  <Welcome setToken={setToken} />
-                )
-              }
-            />
-            <RequireAuth>
+        <div style={{ background: '#fff', borderRadius: 5 }}>
```

Comment:
I'm curious how we all feel about inline styling. I know it's a bit of a hassle to do `makeStyles` or `withStyles` for a couple of styles or a whole new CSS file for HTML Element styles. 

I'm fine with either!

Reply:
For now, I think leaving it as inline styling is good until we make final decisions about the theme/full size of the app container just because it's so easy to see/test. Once we're happy with how it looks, let's move it makeStyles/a css file!